msi/release.sh

```diff
@@ -1,95 +1,233 @@
+                <File Id="PostInstallBat" Source="$BUILD_EXTRA_WINPATH\\post-install.bat" KeyPath="yes" />
+            </Component>
+            <Component Directory="INSTALLFOLDER">
+                <File Id="ReleaseNotes" Source="$BUILD_EXTRA_WINPATH\\ReleaseNotes.html" />
+                <Shortcut Id="ReleaseNotes" Name="Release Notes" Directory="GitProgramMenuFolder" Advertise="yes" />
+            </Component>
+            <Component Directory="INSTALLFOLDER:\\usr\\share\\git\\">
+                <File Id="ReleaseNotes_Css" Source="$BUILD_EXTRA_WINPATH\\usr\\share\\git\\ReleaseNotes.css" />
+            </Component>
+            <Component Directory="INSTALLFOLDER">
+                <RegistryValue Root="HKMU" Key="Software\GitForWindows" Name="CurrentVersion" Type="string" Value="\$(var.Version)" />
+            </Component>
+EOF
+echo "$LIST" |
+sort |
+uniq |
+tr / \\\\ |
+sed -e 's/\(.*\)\\\(.*\)/            <Component Directory="INSTALLFOLDER:\\\1\\">\
+                <File Source="&" \/>\
+            <\/Component>/' \
+        -e 's/^\([^\\]*\)$/            <Component Directory="INSTALLFOLDER">\
+                <File Source="&" \/>\
+            <\/Component>/' \
+        -e 's/\(<File Source="git-bash.exe"[^>]*\) \/>/\1 \/><Shortcut Name="Git Bash" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="PersonalFolder" Advertise="yes" \/>/' \
+        -e 's/\(<File Source="git-cmd.exe"[^>]*\) \/>/\1 \/><Shortcut Name="Git CMD" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="PersonalFolder" Advertise="yes" \/>/' \
+        -e 's/\(<File Source="cmd\\git-gui.exe"[^>]*\) \/>/\1 \/><Shortcut Name="Git GUI" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="PersonalFolder" Advertise="yes" \/>/'
+
+cat <<EOF
+        </ComponentGroup>
+    </Fragment>
+</Wix>
+EOF
+)>GitComponents.wxs
+
+# Make the .msi file
 mkdir -p obj &&
 wix/candle.exe -dVersion="${VERSION%%-*}" \
 	-arch $WIX_ARCH \
-	GitProduct.wxs GitComponents.wxs -o obj\\ \
-	-ext WixUtilExtension &&
+    GitProduct.wxs GitComponents.wxs \
+    GitChooseEnvironmentDlg.wxs GitChooseTerminalDlg.wxs \
+    GitCustomizeLineEndingsDlg.wxs GitLineEndingsDlg.wxs \
+    GitMaintenanceTypeDlg.wxs GitPerformanceOptionsDlg.wxs \
+    -o obj\\ \
+    -ext WixUtilExtension -ext WixTagExtension \
+    -ext WixUIExtension &&
 wix/light.exe \
 	obj/GitProduct.wixobj \
-	obj/GitComponents.wixobj \
-	-o $TARGET -ext WixUtilExtension \
-	-b / -b ../installer &&
-echo "Success! You will find the new .msi at \"$TARGET\"." ||
-die "Could not generate $TARGET"
+    obj/GitComponents.wixobj \
+	obj/GitChooseEnvironmentDlg.wixobj \
+	obj/GitChooseTerminalDlg.wixobj \
+	obj/GitCustomizeLineEndingsDlg.wixobj \
+	obj/GitLineEndingsDlg.wixobj \
+	obj/GitMaintenanceTypeDlg.wixobj \
+	obj/GitPerformanceOptionsDlg.wixobj \
+	-o $TARGET -ext WixUtilExtension -ext WixTagExtension \
+	-ext WixUIExtension -loc GitUI_en-us.wxl \
+	-cultures:en-us \
+	-v \
```

Comment:
Ah, that makes sense. Actually, I just forgot to remove it, but it can be handy during development. I'll probably end up reverting this sooner rather than later and try not to commit this again--it snuck in with the other additions I made.

But that is a good tip! Thanks for the help!